bugbot/rules/inactive_ni_pending.py

```diff
@@ -149,6 +149,7 @@ def get_action_type(bug, inactive_ni):
             bug["priority"] in HIGH_PRIORITY
             or bug["severity"] in HIGH_SEVERITY
             or bug["last_change_time"] >= RECENT_BUG_LIMIT
+            or any(keyword in SECURITY_KEYWORDS for keyword in bug["keywords"])
```

Comment:
NIT: in two other rules we check security keywords in a different approach, it will be nice to unify that (I think the approach that your using in this PR is more readable):

https://github.com/mozilla/bugbot/blob/f6272f4cef19888b28590269079404691c058779/bugbot/rules/crash_small_volume.py#L118

https://github.com/mozilla/bugbot/blob/f6272f4cef19888b28590269079404691c058779/bugbot/rules/security_unhide_dups.py#L51

Reply:
Filed #2099 as a follow-up.